pyanaconda/core/constants.py

```diff
@@ -145,9 +145,13 @@
 # default providers
 GEOLOC_DEFAULT_PROVIDER = GEOLOC_PROVIDER_FEDORA_GEOIP
 GEOLOC_DEFAULT_GEOCODER = GEOLOC_GEOCODER_NOMINATIM
-# timeout (in seconds)
+# how long should the GUI wait for the geolocation thread to finish (in seconds)
+# - GUI starts this count once it fnishes it's initialization
```

Comment:
`finishes`

Reply:
Fixed, thanks! :)

Reply:
Also its, not it's... Didn't want to play spellchecker, but once we're going there, let's do it right :)

Reply:
Fixorzed as well! :)